src/utils.jl

```diff
@@ -245,10 +245,11 @@ end
 # one-liner suggested from ScottPJones
 consumeBOM(buf, pos) = (length(buf) >= 3 && buf[pos] == 0xef && buf[pos + 1] == 0xbb && buf[pos + 2] == 0xbf) ? pos + 3 : pos
 
-if isdefined(Base,:wrap)
-    __wrap(x,pos) = Base.wrap(Array,x,pos)
+if isdefined(Base, :Memory)
```

Comment:
Julia 1.11 also has Base.Memory (but not the recent `wrap` rename) - Will this PR be backwards compatible?

Reply:
the `wrap`->`view` will be back-ported to 1.11

Reply:
It seems rude to break people on the current RC. Can we put in appropriate checks?

Reply:
I can add an extra if for the case that `Memory` and `wrap` is defined